pkg/maps/ctmap/gc/gc.go

```diff
@@ -315,25 +323,18 @@ func (gc *GC) createGCFilter(initialScan bool, restoredEndpoints []*endpoint.End
 			}
 		}
 
+		iter, _ := gc.nodeAddrs.All(gc.db.ReadTxn())
```

Comment:
FIXME: GC needs to watch the Table[NodeAddress] in case it changes and adjust its filter.